### PR TITLE
Loading on edit profile

### PR DESCRIPTION
**Pull request by ervinj1998**
En esta rama se agrego la animacion de carga en el cambio de foto de perfil, tambien se borro variables que no se estan usando actualmente.

### CODE REVIEW

File: src/user/EditProfile.js [hunk 1]
```diff
@@ -118,12 +131,11 @@ clickSubmit = event => {
                             placeholder="John Doe"
                         />
                         {/* If you want the privacy statement it would go here*/}
-        </div> 
+        </div>
                          {/* Next do above; 2X  (email and password) */}
                
         <div className="form-group"> {/* Create the from group "whatever" */}
```
Comment (Hawaiideveloper) [inline src/user/EditProfile.js:137]: form is spelled wrong, but I will correct the code before merging to master, however this appears good enough for alpha

File: src/MainRouter.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import about from './core/About';
+import about from './core/About'; //no lo ocupamos por ahora.
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:3]: this is the support page.

File: src/MainRouter.js [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 import Home from './core/Home';
 import Menu from './core/Menu';
 // import Footer from './core/Footer';
 import Signup from './user/Signup';
 import Signin from './user/Signin';
 import Profile from './user/Profile';
 import Users from './user/Users';
-import SignedOut from './user/YouSignedOut';
+import SignedOut from './user/YouSignedOut';  // no lo ocupamos por ahora
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:11]: sign-out is a needed feature.

File: src/MainRouter.js [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
 import EditProfile from './user/EditProfile'; //usamos el archivo EditProfile
+import PrivateRoute from './auth/PrivateRoute' //importamos la funcion desde el archivo "PrivateRoute.js"
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:13]: creates a workflow for authenticated users only

File: src/MainRouter.js [hunk 5]
```diff
@@ -18,12 +19,16 @@ const MainRouter = () => (
         <Switch>
             <Route exact path="/" component={Home} />
             <Route exact path="/Users" component={Users} />  {/* now we add to the menu next */}
-            <Route exact path="/about" component={about}/>
+
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:22]: missing the about page here

File: src/MainRouter.js [hunk 6]
```diff
@@ -18,12 +19,16 @@ const MainRouter = () => (
...
             <Route exact path="/" component={Home} />
             <Route exact path="/Users" component={Users} />  {/* now we add to the menu next */}
-            <Route exact path="/about" component={about}/>
+    
             <Route exact path="/signup" component={Signup } />
             <Route exact path="/signin" component={Signin} /> 
-            <Route exact path="/signout" component={SignedOut} />
-            <Route exact path="/user/edit/:userId" component={EditProfile} />
-            <Route exact path="/user/:userId" component={Profile} />  
+
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:25]: what happened to signed out?

Comment (Hawaiideveloper) [reply src/MainRouter.js:25]: it no longer gives a page that shows we have successfully signed out.

**Comment by Hawaiideveloper** [comment]
I have made the modifications to keep the features we originally had for users and their confirmations.

@ervinj1998 @manzanarez83  only add or remove what the professor says.  Do not make guesses or remove things that are not removed in the video lessons.

**Comment by Hawaiideveloper** [comment]
please take the time to correct and I will test it like I did the alpha_branch.  Once it is functional and efficient I will merge into alpha_branch.

However at this moment, I do not feel confident about this branch

File: src/MainRouter.js (hunk 4 above)
Comment (ervinj1998) [reply src/MainRouter.js:13]: exacto

**Pull request merged**